build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```

Comment:
Newest version is actually 3.5.2

Reply:
Yeah, that update actually happened right after I submitted the CR for updating the version to 3.5.1.